### PR TITLE
Cleanup title-generation

### PR DESCRIPTION
**Pull request by bkcsoft**

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

**Comment by strk** [comment]
LGTM

File: layouts/partials/header.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <html lang="en">
   <head>
-    <title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ .Title }} - {{ .Site.Title }}{{ end }}</title>
+    <title>{{ if ! .IsHome }}{{ .Title }} - {{ end }}{{ .Site.Title }}</title>
```
Comment (andreynering) [inline layouts/partials/header.html]: Does `!` really work? As far I know it's `not`.

Comment (bkcsoft) [reply layouts/partials/header.html]: Can't find anything on that, but I do find that `if not` works so I'll change it :)

**Pull request merged**